great_expectations/core/util.py

```diff
@@ -98,38 +97,6 @@ def nested_update(
     return d
 
 
-def in_jupyter_notebook():
-    try:
-        shell = get_ipython().__class__.__name__
-        if shell == "ZMQInteractiveShell":
-            return True  # Jupyter notebook or qtconsole
-        elif shell == "TerminalInteractiveShell":
-            return False  # Terminal running IPython
-        else:
-            return False  # Other type (?)
-    except NameError:
-        return False  # Probably standard Python interpreter
-
-
-def determine_progress_bar_method_by_environment() -> Callable:
-    """
-    As tqdm has specific methods for progress bar creation and iteration,
-    we require a utility to determine which method to use.
-
-    If in a Jupyter notebook, we want to use `tqdm.notebook.tqdm`. Otherwise,
-    we default to the standard `tqdm.tqdm`. Please see the docs for more information: https://tqdm.github.io/
-
-    Returns:
-        The appropriate tqdm method for the environment in question.
-    """
-    from tqdm import tqdm
-    from tqdm.notebook import tqdm as tqdm_notebook
-
-    if in_jupyter_notebook():
-        return tqdm_notebook
-    return tqdm
```

Comment:
I would expect removing this will negatively impact the notebook experience which many of our users still actually use.

I think we can still remove the dependency by moving the `ipython` imports inside these methods (and adding a catch for the import error).

Reply:
This method is only ever used in RBP